essencium-backend/src/test/java/de/frachtwerk/essencium/backend/api/data/service/UserServiceStub.java

```diff
@@ -40,12 +44,12 @@ protected <T extends RoleService> LongUserService(
   }
 
   @Override
-  protected @NotNull <E extends UserDto<Long>> TestLongUser convertDtoToEntity(@NotNull E entity) {
+  public @NotNull <E extends UserDto<Long>> UserStub convertDtoToEntity(@NotNull E entity) {
     HashSet<Role> roles =
         entity.getRoles().stream()
             .map(roleService::getByName)
             .collect(Collectors.toCollection(HashSet::new));
-    return TestLongUser.builder()
+    return UserStub.builder()
```

Comment:
Isn't a `.loginDisabled(entity.isLoginDisabled())` missing here?

Reply:
Good catch - it was missing already before the PR changes. I will add it. As I said - using a Stub and not a "real" productive used object is only testing the test implementation. There is no guarantee, that the prod implementation is the same.